specification/cognitiveservices/OpenAI.Inference/models/audio.tsp

```diff
@@ -0,0 +1,206 @@
+
+namespace Azure.OpenAI;
+
+using TypeSpec.Versioning;
+
+@doc("Defines available options for the underlying response format of output transcription information.")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+enum AudioTranscriptionFormat {
+  @doc("Use a response body that is a JSON object containing a single 'text' field for the transcription.")
+  json: "json",
+
+  @doc("""
+    Use a response body that is a JSON object containing transcription text along with timing, segments, and other
+    metadata.
+    """)
+  verbose_json: "verbose_json",
+
+  @doc("Use a response body that is plain text containing the raw, unannotated transcription.")
+  text: "text",
+
+  @doc("Use a response body that is plain text in SubRip (SRT) format that also includes timing information.")
+  srt: "srt",
+
+  @doc("""
+    Use a response body that is plain text in Web Video Text Tracks (VTT) format that also includes timing information.
+    """)
+  vtt: "vtt",
+}
+
+@doc("""
+The configuration information for an audio transcription request.
+""")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+model AudioTranscriptionOptions {
+  @doc("""
+  The audio data to transcribe. This must be the binary content of a file in one of the supported media formats:
+  flac, mp3, mp4, mpeg, mpga, m4a, ogg, wav, webm.
+  """)
+  @projectedName("csharp", "AudioData")
+  file: bytes;
+
+  @doc("""
+  The requested format of the transcription response data, which will influence the content and detail of the result.
+  """)
+  @projectedName("json", "response_format")
+  responseFormat?: AudioTranscriptionFormat;
+
+  @doc("""
+  The primary spoken language of the audio data to be transcribed, supplied as a two-letter ISO-639-1 language code
+  such as 'en' or 'fr'.
+  Providing this known input language is optional but may improve the accuracy and/or latency of transcription.
+  """)
+  language?: string;
+
+  @doc("""
+  An optional hint to guide the model's style or continue from a prior audio segment. The written language of the
+  prompt should match the primary spoken language of the audio data.
+  """)
+  prompt?: string;
+
+  @doc("""
+  The sampling temperature, between 0 and 1.
+  Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic.
+  If set to 0, the model will use log probability to automatically increase the temperature until certain thresholds are hit.
+  """)
+  temperature?: float32;
+
+  @doc("""
+  The model to use for this transcription request.
+  """)
+  // Implementation note: developer-facing specification of deployment or model by clients should be controlled either
+  // via an operation parameter or by this request body field -- but only one of those. This field should be hidden by
+  // clients if operation parameters are used and populated into the request body on an as-needed basis.
+  @projectedName("csharp", "InternalNonAzureModelName")
```

Comment:
Question: Is possible we can do
```
@projectedName("json", "model")
internalNonAzureModelName?: string;
```
So all languages could algin with the same name.